harness/determined/pytorch/_pytorch_trial.py

```diff
@@ -16,8 +16,9 @@
 import torch.distributed as dist
 
 import determined as det
-from determined import layers, pytorch, tensorboard, util, workload
+from determined import layers, pytorch, util, workload
 from determined.horovod import hvd
+from determined.tensorboard.metric_writers.util import is_numerical_scalar
```

Comment:
google style guide for python imports

Reply:
In this file we already have from `determined import util`

Reply:
Ah, I see.  It makes sense to break the rules a bit in that case, but I'd prefer you don't resort to importing symbols directly, because if negatively affects the local understandability of the code.

For example, this line by itself:

```python
is_numerical_scalar(val)
```

_appears_ to be calling a function that is defined within this file.  That's not true, but you have to read all the imports of the file to understand this line.

In this case, can you just move `is_numerical_scalar()` to `det.util`?  I don't see why we need `det.util` and `det.tensorboard.util` and `det.tensorboard.metric_writers.util`.  And I see `is_numeric_scalar()` brings no dependencies with it, so it's harmless to put in the top-level ~trash can~ util module.